### DISCUSSION TITLE
ComfyUI color correction and latent filters

### DISCUSSION THREAD

**Discussion by Chryseus** [Comfy-Org/ComfyUI#2184] [Ideas]
So I came across this interesting [blog post](https://huggingface.co/blog/TimothyAlexisVass/explaining-the-sdxl-latent-space) that made me realise Comfy UI is lacking options to adjust color, I had a brief dig through the code and noticed there appears to be some color correction but it would be nice if such options were exposed to the user as well as additional options showcased in the blog post to enhance image quality.

**Comment by catboxanon** [comment]
Implementation here: https://github.com/Haoming02/comfyui-diffusion-cg
